### PR TITLE
Backfill the concept docs for role assignments with new resources and permissions

### PR DESCRIPTION
**Pull request by alifatolahi-okta**
## Description:
- Some supported resources were not listed and so were newly supported permissions
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-465381](https://oktainc.atlassian.net/browse/OKTA-465381)

### CODE REVIEW

**Review by vasishtajayanti-okta — APPROVED** [review]

**Review by bobhogg-okta — APPROVED** [review]

**Review by williamngo-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 1]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.users.lifecycle.suspend`          | Allows the admin to suspend user access to Okta. When a user is suspended, their user sessions are also cleared.                                      | All Users, all Users within a specific Group |
+| `okta.users.lifecycle.unsuspend`        | Allows the admin to restore user access to Okta                                                                                                      | All Users, all Users within a specific Group |
+| `okta.users.lifecycle.delete`           | Allows the admin to permanently delete user accounts                                                                                                 | All Users, all Users within a specific Group |
+| `okta.users.lifecycle.unlock`           |	Allows the admin to unlock users who have been locked out of Okta                                                                                    | All Users, all Users within a specific Group |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:166]:
```suggestion
| `okta.users.lifecycle.unlock`           |	Allows the admin to unlock users who are locked out of Okta                                                                                    | All Users, all Users within a specific Group |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 2]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
-| `okta.apps.manage`                  | Allows the admin to fully manage Apps and their members in your Okta organization                                                   | All Apps, All apps of specific type, a specific App |
-| `okta.apps.assignment.manage`       | Allows the admin to only manage assignment operations of an App in your Okta org                                                    | All Apps, All apps of specific type, a specific App |
+| Permission type                         | Description                                                                                                                                           | Applicable resource types                    |
+| :-------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------- |
+| `okta.users.manage`                     | Allows the admin to create and manage Users and read all profile and credential information for Users                                                | All Users, all Users within a specific Group |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:152]:
```suggestion
| `okta.users.manage`                     | Allows the admin to create and manage users and read all profile and credential information for users                                                | All Users, all Users within a specific Group |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 3]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.users.create`                     | Allows the admin to create Users. If the admin is also scoped to manage a Group, that admin can add the User to the Group on creation and then manage.| All Groups, a specific Group                 |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:153]:
```suggestion
| `okta.users.create`                     | Allows the admin to create users. If the admin is also scoped to manage a Group, that admin can add the user to the Group on creation and then manage.| All Groups, a specific Group                 |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 4]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.users.read`                       | Allows the admin to read any User's profile and credential information                                                                               | All Users, all Users within a specific Group |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:154]:
```suggestion
| `okta.users.read`                       | Allows the admin to read any user's profile and credential information                                                                               | All Users, all Users within a specific Group |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 5]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.users.credentials.manage`         | Allows the admin to manage only credential lifecycle operations for a User                                                                           | All Users, all Users within a specific Group |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:155]:
```suggestion
| `okta.users.credentials.manage`         | Allows the admin to manage only credential lifecycle operations for a user                                                                           | All Users, all Users within a specific Group |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 6]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.users.lifecycle.delete`           | Allows the admin to permanently delete user accounts                                                                                                 | All Users, all Users within a specific Group |
+| `okta.users.lifecycle.unlock`           |	Allows the admin to unlock users who have been locked out of Okta                                                                                    | All Users, all Users within a specific Group |
+| `okta.users.lifecycle.clearSessions`    | Allows the admin to clear all active Okta sessions and OAuth tokens for a user                                                                       | All Users, all Users within a specific Group |
+| `okta.users.groupMembership.manage`     | Allows the admin to manage a user's group membership (also need `okta.groups.members.manage` to assign to a specific Group)                          | All Users, all Users within a specific Group |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:168]:
```suggestion
| `okta.users.groupMembership.manage`     | Allows the admin to manage a user's group membership (also need `okta.groups.members.manage` to assign to a specific group)                          | All Users, all Users within a specific Group |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 7]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.users.appAssignment.manage`       | Allows the admin to manage a user's app assignment (also need `okta.apps.assignment.manage` to assign to a specific App)                             | All Users, all Users within a specific Group |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:169]:
```suggestion
| `okta.users.appAssignment.manage`       | Allows the admin to manage a user's app assignment (also need `okta.apps.assignment.manage` to assign to a specific app)                             | All Users, all Users within a specific Group |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 8]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.groups.manage`                    | Allows the admin to fully manage Groups in your Okta organization                                                                                    | All Groups, a specific Group                 |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:170]:
```suggestion
| `okta.groups.manage`                    | Allows the admin to fully manage groups in your Okta organization                                                                                    | All Groups, a specific Group                 |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 9]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.groups.create`                    | Allows the admin to create Groups                                                                                                                    | All Groups                                   |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:171]:
```suggestion
| `okta.groups.create`                    | Allows the admin to create groups                                                                                                                    | All Groups                                   |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 10]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.groups.members.manage`            | Allows the admin to only manage member operations in a Group in your Okta org                                                                        | All Groups, a specific Group                 |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:172]:
```suggestion
| `okta.groups.members.manage`            | Allows the admin to only manage member operations in a group in your Okta org                                                                        | All Groups, a specific Group                 |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 11]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.groups.read`                      | Allows the admin to only read information about Groups and their members in your Okta organization                                                   | All Groups, a specific Group                 |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:173]:
```suggestion
| `okta.groups.read`                      | Allows the admin to only read information about groups and their members in your Okta organization                                                   | All Groups, a specific Group                 |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 12]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.groups.appAssignment.manage`      | Allows the admin to manage a Group's app assignment (also need `okta.apps.assignment.manage` to assign to a specific App)                            | All Groups, a specific Group                 |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:174]:
```suggestion
| `okta.groups.appAssignment.manage`      | Allows the admin to manage a group's app assignment (also need `okta.apps.assignment.manage` to assign to a specific app)                            | All Groups, a specific Group                 |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 13]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.apps.read`                        | Allows the admin to only read information about Apps and their members in your Okta organization                                                     | All Apps, All apps of specific type, a specific App |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:175]:
```suggestion
| `okta.apps.read`                        | Allows the admin to only read information about apps and their members in your Okta organization                                                     | All Apps, All apps of specific type, a specific App |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 14]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.apps.manage`                      | Allows the admin to fully manage Apps and their members in your Okta organization                                                                    | All Apps, All apps of specific type, a specific App |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:176]:
```suggestion
| `okta.apps.manage`                      | Allows the admin to fully manage apps and their members in your Okta organization                                                                    | All Apps, All apps of specific type, a specific App |
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 15]
```diff
@@ -144,21 +147,32 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| `okta.apps.assignment.manage`           | Allows the admin to only manage assignment operations of an App in your Okta org                                                                     | All Apps, All apps of specific type, a specific App |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:177]:
```suggestion
| `okta.apps.assignment.manage`           | Allows the admin to only manage assignment operations of an app in your Okta org                                                                     | All Apps, All apps of specific type, a specific App |
```

**Review by susanharper-okta — COMMENTED** [review]
cleaning up some inconsistencies in capitalizations...if it's not specifically talking about an object, it doesn't need to be capitalized. I realize this is probably our fault. Thank you - going to commit these myself....and then approve.

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**